controllers/packagebundle_controller_test.go

```diff
@@ -8,38 +8,58 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/golang/mock/gomock"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 
-	"github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
 	"github.com/aws/eks-anywhere-packages/controllers"
 	"github.com/aws/eks-anywhere-packages/controllers/mocks"
-	bundlefake "github.com/aws/eks-anywhere-packages/pkg/bundle/fake"
 	bundleMocks "github.com/aws/eks-anywhere-packages/pkg/bundle/mocks"
 )
 
 func givenRequest() ctrl.Request {
 	return ctrl.Request{
 		NamespacedName: types.NamespacedName{
 			Name:      "some-bundle",
-			Namespace: v1alpha1.PackageNamespace,
+			Namespace: api.PackageNamespace,
 		},
 	}
 }
 
+func GivenBundle() *api.PackageBundle {
+	return &api.PackageBundle{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v1-22-1001",
```

Comment:
Should we take the bundle name as an argument so this can be used for other tests?